src/specific/s_shell.c

```diff
@@ -133,6 +134,32 @@ int main(int argc, char **argv)
     Log_Init();
 
 #ifdef _WIN32
+    // Enable HiDPI mode in Windows to detect DPI scaling
+    typedef enum PROCESS_DPI_AWARENESS {
+        PROCESS_DPI_UNAWARE = 0,
+        PROCESS_SYSTEM_DPI_AWARE = 1,
+        PROCESS_PER_MONITOR_DPI_AWARE = 2
+    } PROCESS_DPI_AWARENESS;
+
+    HRESULT(WINAPI * SetProcessDpiAwareness)
+    (PROCESS_DPI_AWARENESS dpiAwareness); // Windows 8.1 and later
+    void *shcoreDLL = SDL_LoadObject("SHCORE.DLL");
+    if (shcoreDLL) {
+        SetProcessDpiAwareness =
+            (HRESULT(WINAPI *)(PROCESS_DPI_AWARENESS))SDL_LoadFunction(
+                shcoreDLL, "SetProcessDpiAwareness");
+        if (SetProcessDpiAwareness) {
+            SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
+
+            float ddpi;
+            if (SDL_GetDisplayDPI(0, &ddpi, NULL, NULL)
+                != -1) { // SDL_WINDOWPOS_UNDEFINED is Display 0
+                // When using HiDPI mode, set correct DPI scaling
+                m_Scaling = ddpi / 96.f; // = 0 but not needed for some reason
+            }
```

Comment:
These lines are dead code? `m_Scaling` is never read.

Reply:
Ya it comes out as 0 for some reason. I was using it to multiply by resolution but idk. Maybe I can just delete this part and setting the DPI stuff is enough.